verilog/analysis/verilog_linter.cc

```diff
@@ -210,6 +212,23 @@ LinterConfiguration LinterConfigurationFromFlags() {
   const auto& ruleset = absl::GetFlag(FLAGS_ruleset);
   config.UseRuleSet(ruleset);
 
+  // Read local configuration file
+  std::string content;
+  if (verible::file::GetContents(absl::GetFlag(FLAGS_rules_config), &content)) {
+
+    RuleBundle local_rules_bundle;
+    std::string error;
+    if (local_rules_bundle.ParseConfiguration(content, &error)) {
+      config.UseRuleBundle(local_rules_bundle);
+    } else {
+      LOG(ERROR) << "Unable to fully parse configuration: " << error << std::endl;
+    }
+  } else if(FLAGS_rules_config.IsModified()) {
+    // If flag is modified and  we were unable to open the file, report that
```

Comment:
Does this assumption make sense? It will be similar even when we implement the upward searching

Reply:
If we do upward searching, we need to make it very clear that `rules_config` is interpreted as a file name that will be used to search, and not a one-time-use path.

Reply:
Sure, will do when the search is implemented.